### ISSUE TITLE
feat: users can symlink model binaries

### ISSUE THREAD

**Issue by 0xSage** [janhq/jan#1381]
**Problem**
- Users don't want duplicated model binaries
- Users may want various model.json configuration files for a single model

**Success Criteria**

**Questions:**

1. Should repurpose the model.json > `source_url` property for this? Or something else? I'm out of the loop on this.
1. How to ensure backwards compatibility if so?

```sh
source_url: RELATIVE_PATH_TO_MODEL_BINARY
# versus
source_url: REMOTE_URL
```

**Not in scope yet**
- Files/rag

**Additional context**
- Related: #1010

**Comment by lilskippyy** [comment]
I really need that

**Comment by 0xSage** [comment]
From user: https://superuser.com/questions/1020821/how-can-i-create-a-symbolic-link-on-windows-10

**Comment by namchuai** [comment]
The PR for this ticket has been merged. Please help to verify it @Van-QA . Thank you!

**Comment by louis-jan** [comment]
Since we support moving the January Data Folder, is this issue still relevant? Is the use case where we don't want to put heavy models in C:? If so, users can now freely move model files around disks by setting the January Data Folder. We can deprecate this one since it will complicate model.json and edge cases when users delete a symlink or original model.
cc @imtuyethan @0xSage

**Comment by 0xSage** [comment]
> cases

`model.json` should always be in `jan data folder`, but the `model binary` can be anywhere.

@louis-jan we can close this issue if we end up building #1735  (but isn't it the same thing?).

**Comment by louis-jan** [comment]
@0xSage I've added the impl following model.json update to unblock #1735. 
Users now can use absolute model url in sources.

**Comment by 0xSage** [comment]
> @0xSage I've added the impl following model.json update to unblock #1735. Users now can use absolute model url in sources.

Thanks thats greate, closing this issuel
